### PR TITLE
Misc typo fixes in felix directory

### PR DESCRIPTION
**Pull request by Akasurde**
## Description
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: felix/bpf-gpl/include/libbpf/src/btf.c [hunk 1]
```diff
@@ -2828,7 +2828,7 @@ static int btf_dedup_remap_types(struct btf_dedup *d);
  * types, in general, can form graphs containing cycles, not just DAGs.
  *
  * While algorithm does deduplication, it also merges and resolves type
- * information (unless disabled throught `struct btf_opts`), whenever possible.
+ * information (unless disabled thought `struct btf_opts`), whenever possible.
```
Comment (neiljerram) [inline felix/bpf-gpl/include/libbpf/src/btf.c:2831]: Should be "through"

Comment (neiljerram) [reply felix/bpf-gpl/include/libbpf/src/btf.c:2831]: Also BTW the upstream for felix/bpf-gpl/include/libbpf is in the kernel: see the README at https://github.com/libbpf/libbpf.  Hence these fixes are likely to be lost if we sync with a newer version of upstream, and you may like to look at contributing the fixes upstream as well.

**Review by neiljerram — CHANGES_REQUESTED** [review]
Thanks again.  One correction, and one note about libbpf.

File: felix/bpf-gpl/include/libbpf/src/btf.c (hunk 1 above)
Comment (Akasurde) [reply felix/bpf-gpl/include/libbpf/src/btf.c:2831]: Just as I thought. I will close this PR and will open against the libbpf. Thanks for the advice.

**Review by neiljerram — APPROVED** [review]

**Comment by neiljerram** [comment]
(Ignoring known and unrelated CI flakes)

**Pull request merged**